src/CMakeLists.txt

```diff
@@ -64,6 +64,9 @@ endif()
 target_link_libraries(qtadvanceddocking PUBLIC Qt${QT_VERSION_MAJOR}::Core 
                                                Qt${QT_VERSION_MAJOR}::Gui 
                                                Qt${QT_VERSION_MAJOR}::Widgets)
+if (UNIX AND NOT APPLE)
+  target_link_libraries(qtadvanceddocking PUBLIC xcb)
```

Comment:
maybe you also need a find_library call for this library. This may fail silently if the xcb target is not created